builtins/packagedcode_msitools-linux/lib-src/bats-core-ce5ca2802fabe5dc38393240cd40e20f8928d3b0.zip.ABOUT

```diff
@@ -0,0 +1,9 @@
+about_resource: bats-core-ce5ca2802fabe5dc38393240cd40e20f8928d3b0.zip
```

Comment:
Are you sure this is needed just to build msitools?

Reply:
The meson buildscript requires this. A git submodule of bats-core is used during build time: https://gitlab.gnome.org/GNOME/msitools/-/tree/master/subprojects